lang_scala/parsing/Lexer_scala.mll

```diff
@@ -507,15 +507,13 @@ and in_interpolated_triple = parse
 (* Rule comment *)
 (*****************************************************************************)
 
-and comment buf = parse
-  | "*/"    { Buffer.add_string buf (tok lexbuf) }
-  (* noteopti: *)
```

Comment:
This was a little optimization to avoid to call 50 times Buffer.add_string with a single char.
With your '_ { ... ' you match only one character at a time.